include/dsn/dist/replication/replication_ddl_client.h

```diff
@@ -169,6 +169,11 @@ class replication_ddl_client
 
     dsn::error_code ddd_diagnose(gpid pid, std::vector<ddd_partition_info> &ddd_partitions);
 
+    void
+    query_disk_info(const std::vector<dsn::rpc_address> &targets,
+                    /*out*/ std::map<dsn::rpc_address, error_with<query_disk_info_response>> &resps,
```

Comment:
If output-only parameters are used, they should appear after input parameters.
-- https://google.github.io/styleguide/cppguide.html#Output_Parameters